### ISSUE TITLE
strlen() expects parameter 1 to be string, array given

### ISSUE THREAD

**Issue by andrewwindfall** [filp/whoops#627]
Hello,
I get the following error on any laravel error. I have recently upgraded from 5.6 to 5.8, but I cannot say for certain if the problem started after the upgrade. My composer.json looks like:

(1/1) ErrorException
strlen() expects parameter 1 to be string, array given
--
in PrettyPageHandler.php line 708

```

"require": {
        "php": ">=7.1.3",
        "fideloper/proxy": "~4.0",
        "jgrossi/corcel": "^2.7",
        "laravel/framework": "5.8.*",
        "laravel/tinker": "~1.0"
    },
    "require-dev": {
        "filp/whoops": "~2.0",
        "fzaninotto/faker": "~1.4",
        "mockery/mockery": "~1.0",
        "phpunit/phpunit": "^7.5",
        "symfony/thanks": "^1.0"
    },

```

Php info:
`php 7.2.14`

**Comment by staabm** [comment]
Do you have a stacktrace and/or file/lineno of the issue/warning in question?

**Comment by andrewwindfall** [comment]
Here is the full stack
```

(1/1) ErrorExceptionstrlen() expects parameter 1 to be string, array given
--
in PrettyPageHandler.php line 708
at HandleExceptions->handleError(2, 'strlen() expects parameter 1 to be string, array given', '/Users/***/Desktop/MAMP/se/vendor/filp/whoops/src/Whoops/Handler/PrettyPageHandler.php', 708, array('superGlobal' => array('PHP_FCGI_CHILDREN' => '4', 'PWD' => '/Applications/MAMP/fcgi-bin', 'SHLVL' => '0', 'PHP_FCGI_MAX_REQUESTS' => '200', '__CF_USER_TEXT_ENCODING' => '0x1F5:0x0:0x0', 'ORIG_SCRIPT_NAME' => '/fcgi-bin/php7.2.14.fcgi', 'ORIG_PATH_TRANSLATED' => '/Users/***/Desktop/MAMP/se/public/index.php', 'ORIG_PATH_INFO' => '/index.php', 'ORIG_SCRIPT_FILENAME' => '/Applications/MAMP/fcgi-bin/php7.2.14.fcgi', 'SCRIPT_NAME' => '/index.php', 'REQUEST_URI' => '/dashboard/deals', 'QUERY_STRING' => '', 'REQUEST_METHOD' => 'POST', 'SERVER_PROTOCOL' => 'HTTP/1.1', 'GATEWAY_INTERFACE' => 'CGI/1.1', 'REDIRECT_URL' => '/index.php', 'REMOTE_PORT' => '51489', 'SCRIPT_FILENAME' => '/Users/***/Desktop/MAMP/se/public/index.php', 'SERVER_ADMIN' => 'you@example.com', 'DOCUMENT_ROOT' => '/Users/***/Desktop/MAMP/se/public', 'REMOTE_ADDR' => '::1', 'SERVER_PORT' => '80', 'SERVER_ADDR' => '::1', 'SERVER_NAME' => 'semt.local', 'SERVER_SOFTWARE' => 'Apache', 'SERVER_SIGNATURE' => '', 'PATH' => '/usr/bin:/bin:/usr/sbin:/sbin', 'HTTP_COOKIE' => '_ga=GA1.2.10422467.1557862642; _gid=GA1.2.1514529248.1557862642; remember_web_59ba36addc2b2f9401580f014c7f58ea4e30989d=eyJpdiI6IndYMGRObjRVYXF6bENxSVwvSVdtXC9lQT09IiwidmFsdWUiOiIyM2lVZFlpT01pUWQzNTFzekRidjZrQXMyXC9uejFETjhrZ1ZwUmRENWUybWFocG5OWkl1Z21FVElHbGc3eVRRSXZhQXU4eFJaTXBjXC94Um4xVW5aZFRoQm5vZG9hOGFxV3lFSG9ra2U1Qk9laEVZYkwxa0krK3NvSDFSUUZIRzFpd3JsNk1QcjZWUVNWajl2emZmWDdieUVhXC85Y2dCSEFrWVZnZzRySDlmNlk9IiwibWFjIjoiOTBhNjhjMDgwMGE5ZjMzZjI1ZDg4NGI3MDc2YTM0MTViYjg1YzhhNmJlOTRhZjk2MGZjZDViNjE4Y2MxZDBiMCJ9; XSRF-TOKEN=eyJpdiI6IjdVZWlBamlcL1V0VGE5dDQ4UlM2aWZ3PT0iLCJ2YWx1ZSI6IkV4SWtQS3JlTGlMOXVHNkFEc1ZYUldqc3Z2M2R1TVYrMklHXC9Jbkg0MlFjT0ppNytkVlowZVlxeG9aMnFzR0VRIiwibWFjIjoiMjE5NDM3MjU4ZWRhYTRkNGJjYTUzZDk1MTkyYWI4Zjc4NGM5N2E3NTYwNTVhNmFhMmQyNDQ0ODA4ZjQ1ZjFlYSJ9; laravel_session=eyJpdiI6ImRuSnhDbG84Ukh2c1IzejFZempSTHc9PSIsInZhbHVlIjoiTkc3NlVDcStmSDd4YklnOXV2MXI4VXpnVDFseHh3WEZRaHZJZE8xcXNRa0R0SEM3UnZMdTduTzEzT2tXSWhMbSIsIm1hYyI6ImIzNWIzOWYwMmZlOGZhOGU3MjhkODlhZjk3Zjk4MmQ5ZWQwMTk0MDM5OTBhYjU0ODIwNDE0ZmExZDdkZTRkZDYifQ%3D%3D', 'HTTP_ACCEPT_LANGUAGE' => 'en-US,en;q=0.9,la;q=0.8', 'HTTP_ACCEPT_ENCODING' => 'gzip, deflate', 'HTTP_REFERER' => 'http://semt.local/dashboard/deals/create', 'HTTP_ACCEPT' => 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3', 'CONTENT_TYPE' => 'multipart/form-data; boundary=----WebKitFormBoundary85glw2YxrKcvLUfx', 'HTTP_ORIGIN' => 'http://semt.local', 'HTTP_USER_AGENT' => 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.131 Safari/537.36', 'HTTP_UPGRADE_INSECURE_REQUESTS' => '1', 'HTTP_CACHE_CONTROL' => 'max-age=0', 'CONTENT_LENGTH' => '481814', 'HTTP_CONNECTION' => 'keep-alive', 'HTTP_HOST' => 'semt.local', 'REDIRECT_STATUS' => '200', 'REDIRECT_HANDLER' => 'php-fastcgi', 'REDIRECT_REDIRECT_STATUS' => '200', 'FCGI_ROLE' => 'RESPONDER', 'PHP_SELF' => '/index.php', 'REQUEST_TIME_FLOAT' => 1558031874.078983, 'REQUEST_TIME' => 1558031874, 'argv' =>array(), 'argc' => 0, 'APP_NAME' => 'Laravel', 'APP_ENV' => 'local', 'APP_KEY' => '***=', 'APP_DEBUG' => 'true', 'APP_LOG_LEVEL' => 'debug', 'APP_URL' => 'http://localhost', 'DB_CONNECTION' => 'mysql', 'DB_HOST' => '***', 'DB_PORT' => '3306', 'DB_DATABASE' => '***', 'DB_USERNAME' => '***', 'DB_PASSWORD' => '***', 'DB_DATABASE_STATE' => '***', 'DB_DATABASE_BLOG' => '***', 'DB_USERNAME_BLOG' => '***', 'DB_PASSWORD_BLOG' => '***', 'DB_HOST_EVENTS' => '***', 'DB_DATABASE_EVENTS' => '***', 'DB_USERNAME_EVENTS' => '***', 'DB_PASSWORD_EVENTS' => 'aMb6d%2q[b!v', 'BROADCAST_DRIVER' => 'log', 'CACHE_DRIVER' => 'file', 'SESSION_DRIVER' => 'file', 'SESSION_LIFETIME' => '120', 'QUEUE_DRIVER' => 'sync', 'REDIS_HOST' => '127.0.0.1', 'REDIS_PASSWORD' => 'null', 'REDIS_PORT' => '6379', 'MAIL_DRIVER' => 'smtp', 'MAIL_HOST' => 'smtp.mailtrap.io', 'MAIL_PORT' => '2525', 'MAIL_USERNAME' => 'null', 'MAIL_PASSWORD' => 'null', 'MAIL_ENCRYPTION' => 'null', 'PUSHER_APP_ID' => '', 'PUSHER_APP_KEY' => '', 'PUSHER_APP_SECRET' => '', 'PUSHER_APP_CLUSTER' => 'mt1'), 'superGlobalName' => '_SERVER', 'blacklisted' => array('PHP_AUTH_PW', 'PHP_FCGI_CHILDREN', 'PWD', 'SHLVL', 'PHP_FCGI_MAX_REQUESTS', '__CF_USER_TEXT_ENCODING', 'ORIG_SCRIPT_NAME', 'ORIG_PATH_TRANSLATED', 'ORIG_PATH_INFO', 'ORIG_SCRIPT_FILENAME', 'SCRIPT_NAME', 'REQUEST_URI', 'QUERY_STRING', 'REQUEST_METHOD', 'SERVER_PROTOCOL', 'GATEWAY_INTERFACE', 'REDIRECT_URL', 'REMOTE_PORT', 'SCRIPT_FILENAME', 'SERVER_ADMIN', 'DOCUMENT_ROOT', 'REMOTE_ADDR', 'SERVER_PORT', 'SERVER_ADDR', 'SERVER_NAME', 'SERVER_SOFTWARE', 'SERVER_SIGNATURE', 'PATH', 'HTTP_COOKIE', 'HTTP_ACCEPT_LANGUAGE', 'HTTP_ACCEPT_ENCODING', 'HTTP_REFERER', 'HTTP_ACCEPT', 'CONTENT_TYPE', 'HTTP_ORIGIN', 'HTTP_USER_AGENT', 'HTTP_UPGRADE_INSECURE_REQUESTS', 'HTTP_CACHE_CONTROL', 'CONTENT_LENGTH', 'HTTP_CONNECTION', 'HTTP_HOST', 'REDIRECT_STATUS', 'REDIRECT_HANDLER', 'REDIRECT_REDIRECT_STATUS', 'FCGI_ROLE', 'PHP_SELF', 'REQUEST_TIME_FLOAT', 'REQUEST_TIME', 'argv', 'argc', 'APP_NAME', 'APP_ENV', 'APP_KEY', 'APP_DEBUG', 'APP_LOG_LEVEL', 'APP_URL', 'DB_CONNECTION', 'DB_HOST', 'DB_PORT', 'DB_DATABASE', 'DB_USERNAME', 'DB_PASSWORD', 'DB_DATABASE_STATE', 'DB_DATABASE_BLOG', 'DB_USERNAME_BLOG', 'DB_PASSWORD_BLOG', 'DB_HOST_EVENTS', 'DB_DATABASE_EVENTS', 'DB_USERNAME_EVENTS', 'DB_PASSWORD_EVENTS', 'BROADCAST_DRIVER', 'CACHE_DRIVER', 'SESSION_DRIVER', 'SESSION_LIFETIME', 'QUEUE_DRIVER', 'REDIS_HOST', 'REDIS_PASSWORD', 'REDIS_PORT', 'MAIL_DRIVER', 'MAIL_HOST', 'MAIL_PORT', 'MAIL_USERNAME', 'MAIL_PASSWORD', 'MAIL_ENCRYPTION', 'PUSHER_APP_ID', 'PUSHER_APP_KEY', 'PUSHER_APP_SECRET', 'PUSHER_APP_CLUSTER'), 'values' => array('PHP_FCGI_CHILDREN' => '*', 'PWD' => '***************************', 'SHLVL' => '*', 'PHP_FCGI_MAX_REQUESTS' => '***', '__CF_USER_TEXT_ENCODING' => '*************', 'ORIG_SCRIPT_NAME' => '************************', 'ORIG_PATH_TRANSLATED' => '**********************************************', 'ORIG_PATH_INFO' => '**********', 'ORIG_SCRIPT_FILENAME' => '******************************************', 'SCRIPT_NAME' => '**********', 'REQUEST_URI' => '****************', 'QUERY_STRING' => '', 'REQUEST_METHOD' => '****', 'SERVER_PROTOCOL' => '********', 'GATEWAY_INTERFACE' => '*******', 'REDIRECT_URL' => '**********', 'REMOTE_PORT' => '*****', 'SCRIPT_FILENAME' => '**********************************************', 'SERVER_ADMIN' => '***************', 'DOCUMENT_ROOT' => '************************************', 'REMOTE_ADDR' => '***', 'SERVER_PORT' => '**', 'SERVER_ADDR' => '***', 'SERVER_NAME' => '**********', 'SERVER_SOFTWARE' => '******', 'SERVER_SIGNATURE' => '', 'PATH' => '*****************************', 'HTTP_COOKIE' => '**************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************', 'HTTP_ACCEPT_LANGUAGE' => '***********************', 'HTTP_ACCEPT_ENCODING' => '*************', 'HTTP_REFERER' => '****************************************', 'HTTP_ACCEPT' => '**********************************************************************************************************************', 'CONTENT_TYPE' => '********************************************************************', 'HTTP_ORIGIN' => '*****************', 'HTTP_USER_AGENT' => '*************************************************************************************************************************', 'HTTP_UPGRADE_INSECURE_REQUESTS' => '*', 'HTTP_CACHE_CONTROL' => '*********', 'CONTENT_LENGTH' => '******', 'HTTP_CONNECTION' => '**********', 'HTTP_HOST' => '**********', 'REDIRECT_STATUS' => '***', 'REDIRECT_HANDLER' => '***********', 'REDIRECT_REDIRECT_STATUS' => '***', 'FCGI_ROLE' => '*********', 'PHP_SELF' => '**********', 'REQUEST_TIME_FLOAT' => '**************', 'REQUEST_TIME' => '**********', 'argv' => array(), 'argc' => 0, 'APP_NAME' => 'Laravel', 'APP_ENV' => 'local', 'APP_KEY' => 'base64:bDydtNb4m5x6vpcwgebv73MG+/BulVNBIblkvlSTWmM=', 'APP_DEBUG' => 'true', 'APP_LOG_LEVEL' => 'debug', 'APP_URL' => 'http://localhost', 'DB_CONNECTION' => 'mysql', 'DB_HOST' => '108.179.219.95', 'DB_PORT' => '3306', 'DB_DATABASE' => '***', 'DB_USERNAME' => '***', 'DB_PASSWORD' => '***', 'DB_DATABASE_STATE' => '***', 'DB_DATABASE_BLOG' => '***', 'DB_USERNAME_BLOG' => '***', 'DB_PASSWORD_BLOG' => '***', 'DB_HOST_EVENTS' => '***', 'DB_DATABASE_EVENTS' => '***', 'DB_USERNAME_EVENTS' => '***', 'DB_PASSWORD_EVENTS' => '***', 'BROADCAST_DRIVER' => 'log', 'CACHE_DRIVER' => 'file', 'SESSION_DRIVER' => 'file', 'SESSION_LIFETIME' => '120', 'QUEUE_DRIVER' => 'sync', 'REDIS_HOST' => '127.0.0.1', 'REDIS_PASSWORD' => 'null', 'REDIS_PORT' => '6379', 'MAIL_DRIVER' => 'smtp', 'MAIL_HOST' => 'smtp.mailtrap.io', 'MAIL_PORT' => '2525', 'MAIL_USERNAME' => 'null', 'MAIL_PASSWORD' => 'null', 'MAIL_ENCRYPTION' => 'null', 'PUSHER_APP_ID' => '', 'PUSHER_APP_KEY' => '', 'PUSHER_APP_SECRET' => '', 'PUSHER_APP_CLUSTER' => 'mt1'), 'key' => 'argv'))
at strlen(array())in PrettyPageHandler.php line 708
at PrettyPageHandler->masked(array('PHP_FCGI_CHILDREN' => '4', 'PWD' => '/Applications/MAMP/fcgi-bin', 'SHLVL' => '0', 'PHP_FCGI_MAX_REQUESTS' => '200', '__CF_USER_TEXT_ENCODING' => '0x1F5:0x0:0x0', 'ORIG_SCRIPT_NAME' => '/fcgi-bin/php7.2.14.fcgi', 'ORIG_PATH_TRANSLATED' => '/Users/andrew/Desktop/MAMP/se/public/index.php', 'ORIG_PATH_INFO' => '/index.php', 'ORIG_SCRIPT_FILENAME' => '/Applications/MAMP/fcgi-bin/php7.2.14.fcgi', 'SCRIPT_NAME' => '/index.php', 'REQUEST_URI' => '/dashboard/deals', 'QUERY_STRING' => '', 'REQUEST_METHOD' => 'POST', 'SERVER_PROTOCOL' => 'HTTP/1.1', 'GATEWAY_INTERFACE' => 'CGI/1.1', 'REDIRECT_URL' => '/index.php', 'REMOTE_PORT' => '51489', 'SCRIPT_FILENAME' => '/Users/andrew/Desktop/MAMP/se/public/index.php', 'SERVER_ADMIN' => 'you@example.com', 'DOCUMENT_ROOT' => '/Users/***/Desktop/MAMP/se/public', 'REMOTE_ADDR' => '::1', 'SERVER_PORT' => '80', 'SERVER_ADDR' => '::1', 'SERVER_NAME' => 'semt.local', 'SERVER_SOFTWARE' => 'Apache', 'SERVER_SIGNATURE' => '', 'PATH' => '/usr/bin:/bin:/usr/sbin:/sbin', 'HTTP_COOKIE' => '_ga=GA1.2.10422467.1557862642; _gid=GA1.2.1514529248.1557862642; remember_web_59ba36addc2b2f9401580f014c7f58ea4e30989d=eyJpdiI6IndYMGRObjRVYXF6bENxSVwvSVdtXC9lQT09IiwidmFsdWUiOiIyM2lVZFlpT01pUWQzNTFzekRidjZrQXMyXC9uejFETjhrZ1ZwUmRENWUybWFocG5OWkl1Z21FVElHbGc3eVRRSXZhQXU4eFJaTXBjXC94Um4xVW5aZFRoQm5vZG9hOGFxV3lFSG9ra2U1Qk9laEVZYkwxa0krK3NvSDFSUUZIRzFpd3JsNk1QcjZWUVNWajl2emZmWDdieUVhXC85Y2dCSEFrWVZnZzRySDlmNlk9IiwibWFjIjoiOTBhNjhjMDgwMGE5ZjMzZjI1ZDg4NGI3MDc2YTM0MTViYjg1YzhhNmJlOTRhZjk2MGZjZDViNjE4Y2MxZDBiMCJ9; XSRF-TOKEN=eyJpdiI6IjdVZWlBamlcL1V0VGE5dDQ4UlM2aWZ3PT0iLCJ2YWx1ZSI6IkV4SWtQS3JlTGlMOXVHNkFEc1ZYUldqc3Z2M2R1TVYrMklHXC9Jbkg0MlFjT0ppNytkVlowZVlxeG9aMnFzR0VRIiwibWFjIjoiMjE5NDM3MjU4ZWRhYTRkNGJjYTUzZDk1MTkyYWI4Zjc4NGM5N2E3NTYwNTVhNmFhMmQyNDQ0ODA4ZjQ1ZjFlYSJ9; laravel_session=eyJpdiI6ImRuSnhDbG84Ukh2c1IzejFZempSTHc9PSIsInZhbHVlIjoiTkc3NlVDcStmSDd4YklnOXV2MXI4VXpnVDFseHh3WEZRaHZJZE8xcXNRa0R0SEM3UnZMdTduTzEzT2tXSWhMbSIsIm1hYyI6ImIzNWIzOWYwMmZlOGZhOGU3MjhkODlhZjk3Zjk4MmQ5ZWQwMTk0MDM5OTBhYjU0ODIwNDE0ZmExZDdkZTRkZDYifQ%3D%3D', 'HTTP_ACCEPT_LANGUAGE' => 'en-US,en;q=0.9,la;q=0.8', 'HTTP_ACCEPT_ENCODING' => 'gzip, deflate', 'HTTP_REFERER' => 'http://semt.local/dashboard/deals/create', 'HTTP_ACCEPT' => 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3', 'CONTENT_TYPE' => 'multipart/form-data; boundary=----WebKitFormBoundary85glw2YxrKcvLUfx', 'HTTP_ORIGIN' => 'http://semt.local', 'HTTP_USER_AGENT' => 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.131 Safari/537.36', 'HTTP_UPGRADE_INSECURE_REQUESTS' => '1', 'HTTP_CACHE_CONTROL' => 'max-age=0', 'CONTENT_LENGTH' => '481814', 'HTTP_CONNECTION' => 'keep-alive', 'HTTP_HOST' => 'semt.local', 'REDIRECT_STATUS' => '200', 'REDIRECT_HANDLER' => 'php-fastcgi', 'REDIRECT_REDIRECT_STATUS' => '200', 'FCGI_ROLE' => 'RESPONDER', 'PHP_SELF' => '/index.php', 'REQUEST_TIME_FLOAT' => 1558031874.078983, 'REQUEST_TIME' => 1558031874, 'argv' =>array(), 'argc' => 0, 'APP_NAME' => 'Laravel', 'APP_ENV' => 'local', 'APP_KEY' => '***', 'APP_DEBUG' => 'true', 'APP_LOG_LEVEL' => 'debug', 'APP_URL' => 'http://localhost', 'DB_CONNECTION' => 'mysql', 'DB_HOST' => '108.179.219.95', 'DB_PORT' => '3306', 'DB_DATABASE' => '***', 'DB_USERNAME' => '***', 'DB_PASSWORD' => 'G,q=[^AT;AfC', 'DB_DATABASE_STATE' => '***', 'DB_DATABASE_BLOG' => '***', 'DB_USERNAME_BLOG' => '***', 'DB_PASSWORD_BLOG' => 'V[?Q}CV~3B-*', 'DB_HOST_EVENTS' => '108.179.219.95', 'DB_DATABASE_EVENTS' => '***', 'DB_USERNAME_EVENTS' => '***', 'DB_PASSWORD_EVENTS' => 'aMb6d%2q[b!v', 'BROADCAST_DRIVER' => 'log', 'CACHE_DRIVER' => 'file', 'SESSION_DRIVER' => 'file', 'SESSION_LIFETIME' => '120', 'QUEUE_DRIVER' => 'sync', 'REDIS_HOST' => '127.0.0.1', 'REDIS_PASSWORD' => 'null', 'REDIS_PORT' => '6379', 'MAIL_DRIVER' => 'smtp', 'MAIL_HOST' => 'smtp.mailtrap.io', 'MAIL_PORT' => '2525', 'MAIL_USERNAME' => 'null', 'MAIL_PASSWORD' => 'null', 'MAIL_ENCRYPTION' => 'null', 'PUSHER_APP_ID' => '', 'PUSHER_APP_KEY' => '', 'PUSHER_APP_SECRET' => '', 'PUSHER_APP_CLUSTER' => 'mt1'), '_SERVER')in PrettyPageHandler.php line 230
at PrettyPageHandler->handle(object(FatalThrowableError))in Run.php line 272
at Run->handleException(object(FatalThrowableError))in Handler.php line 339
at Handler->renderExceptionWithWhoops(object(FatalThrowableError))in Handler.php line 318
at Handler->renderExceptionContent(object(FatalThrowableError))in Handler.php line 302
at Handler->convertExceptionToResponse(object(FatalThrowableError))in Handler.php line 281
at Handler->prepareResponse(object(Request), object(FatalThrowableError))in Handler.php line 188
at Handler->render(object(Request), object(FatalThrowableError))in Handler.php line 56
at Handler->render(object(Request), object(FatalThrowableError))in Pipeline.php line 83
at Pipeline->handleException(object(Request), object(FatalThrowableError))in Pipeline.php line 34
at Pipeline->Illuminate\Routing\{closure}(object(Request))in SubstituteBindings.php line 41
at SubstituteBindings->handle(object(Request), object(Closure))in Pipeline.php line 163
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request))in Authenticate.php line 43
at Authenticate->handle(object(Request), object(Closure))in Pipeline.php line 163
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request))in VerifyCsrfToken.php line 75
at VerifyCsrfToken->handle(object(Request), object(Closure))in Pipeline.php line 163
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request))in ShareErrorsFromSession.php line 49
at ShareErrorsFromSession->handle(object(Request), object(Closure))in Pipeline.php line 163
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request))in StartSession.php line 56
at StartSession->handle(object(Request), object(Closure))in Pipeline.php line 163
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request))in AddQueuedCookiesToResponse.php line 37
at AddQueuedCookiesToResponse->handle(object(Request), object(Closure))in Pipeline.php line 163
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request))in EncryptCookies.php line 66
at EncryptCookies->handle(object(Request), object(Closure))in Pipeline.php line 163
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request))in Pipeline.php line 104
at Pipeline->then(object(Closure))in Router.php line 682
at Router->runRouteWithinStack(object(Route), object(Request))in Router.php line 657
at Router->runRoute(object(Request), object(Route))in Router.php line 623
at Router->dispatchToRoute(object(Request))in Router.php line 612
at Router->dispatch(object(Request))in Kernel.php line 176
at Kernel->Illuminate\Foundation\Http\{closure}(object(Request))in Pipeline.php line 30
at Pipeline->Illuminate\Routing\{closure}(object(Request))in TrustProxies.php line 57
at TrustProxies->handle(object(Request), object(Closure))in Pipeline.php line 163
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request))in TransformsRequest.php line 21
at TransformsRequest->handle(object(Request), object(Closure))in Pipeline.php line 163
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request))in TransformsRequest.php line 21
at TransformsRequest->handle(object(Request), object(Closure))in Pipeline.php line 163
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request))in ValidatePostSize.php line 27
at ValidatePostSize->handle(object(Request), object(Closure))in Pipeline.php line 163
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request))in CheckForMaintenanceMode.php line 62
at CheckForMaintenanceMode->handle(object(Request), object(Closure))in Pipeline.php line 163
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request))in Pipeline.php line 104
at Pipeline->then(object(Closure))in Kernel.php line 151
at Kernel->sendRequestThroughRouter(object(Request))in Kernel.php line 116
at Kernel->handle(object(Request))in index.php line 55

```

**Comment by staabm** [comment]
Could you please give the fix at https://github.com/filp/whoops/pull/628 a try?

**Comment by andrewwindfall** [comment]
That worked perfectly! Thanks so much!

**Comment by williamdes** [comment]
Fixed the issue for me, #628 is merged now :tada:
